lib/v2/daily/utils.js

```diff
@@ -0,0 +1,40 @@
+const dayjs = require('dayjs');
+const got = require('@/utils/got');
+
+const getData = async (graphqlQuery) =>
+    (
+        await got
+            .post('https://app.daily.dev/api/graphql', {
+                json: graphqlQuery,
+            })
+            .json()
+    ).data.page.edges;
+
+const getList = (data) =>
+    data.map((value) => {
+        const { id, title, image, permalink, summary, createdAt, numUpvotes, author, tags, numComments } = value.node;
+        const pubDate = dayjs(createdAt);
+        return {
+            id,
+            title,
+            link: permalink,
+            description: summary,
+            author: author?.name,
+            itunes_item_image: image,
+            pubDate,
+            upvotes: numUpvotes,
+            comments: numComments,
+            category: tags,
+        };
+    });
+
+const getRedirectedLink = (data) =>
+    Promise.all(
+        data.map(async (v) => {
+            const instance = got.extend({ followRedirect: true });
```

Comment:
`followRedirect` is `true` by default.
https://github.com/sindresorhus/got/tree/v11.8.6#followredirect